github-cache/main.py

```
@@ -0,0 +1,112 @@
+
+GITHUB_API_KEY = os.environ.get("GITHUB_API_KEY")
+app = FastAPI(root_path="/github/v1", title="Gamemaster AI Github-Proxy")
+
+
+async def fetch_contributors() -> list:
+    headers = {}
+    if GITHUB_API_KEY:
+        headers["Authorization"] = f"token {GITHUB_API_KEY}"
+
+    async with aiohttp.ClientSession() as session:
+        async with session.get("https://api.github.com/repos/bjoern-buettner/roleplay-ai/contributors", headers=headers) as response:
```

Ruff (0.12.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)